file_formats/ie_formats/itm_v1.htm

```diff
@@ -735,6 +735,13 @@
     </table>
     <br />
     * In BG1, this flag does not affect whether the item can be sold.<br />
+    ** Forces Two-Handed Weapon style, disables off-hand attacks and equipped effects.<br />
+    *** Disables off-hand attacks, equipped effects and Two-Weapon style (inventory still displays Two-Weapon style modifiers, but they are not included in the calculations).<br />
+    **** It does not disable equipped effects of the off-hand (weapon or shield). In particular:
```

Comment:
Isn't this only true if this item is equipped in the off hand? The name of the bit and the description don't quite match.

Reply:
> Isn't this only true if this item is equipped in the off hand? The name of the bit and the description don't quite match.

All of these flags only matter if they are on the main hand weapon.

Reply:
Still, if it does what the description says, the bit name needs changing.